hvpy/api_groups/screenshots/take_screenshot.py

```diff
@@ -0,0 +1,111 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+from hvpy.utils import convert_date_to_isoformat
+
+
+class takeScreenshotInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the takeScreenshot API.
+
+    Attributes
+    ----------
+    {Shared}
+    date : datetime.datetime
+        Desired date/time of the image.
+
+    imageScale : float
+        Image scale in arcseconds per pixel.
+
```

Comment:
I think these line breaks should be removed.

Reply:
Do you mean the changes in the docstring you did #45?